playwright.config.js

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable semi */
```

Comment:
nit: why not just remove the 2 semi-colons and avoid having to disable the ESLint rule?